.github/workflows/ci.yml

```diff
@@ -55,12 +55,12 @@ jobs:
 
       - name: Test
         env:
-          BUILD_TAGS: example,local,ecs
+          BUILD_TAGS: example,local
         run: make -f builder.Makefile test
 
       - name: Build
         env:
-          BUILD_TAGS: example,local,ecs
+          BUILD_TAGS: example,local,e2e
```

Comment:
Not for this PR but I think we need to rename the steps in the CI workflow. It's a bit weird to do end to end testing in the "Build" step.

I noticed this because we're setting e2e as a build tag here which sounded wrong for build.

Reply:
updated name